dipy/core/benchmarks/bench_sphere.py

```diff
@@ -0,0 +1,118 @@
+""" Benchmarks for sphere
+
+Run all benchmarks with::
+
+    import dipy.core as dipycore
+    dipycore.bench()
+
+With Pytest, Run this benchmark with:
+
+    pytest -svv -c bench.ini /path/to/bench_sphere.py
```

Comment:
Not sure the the command works correctly. May be it's just a local issue I'm having.